Libplanet.Tools/Mpt.cs

```diff
@@ -157,6 +129,14 @@ public class Mpt
             string uri,
             [FromService] IConfigurationService<ToolConfiguration> configurationService)
         {
+            if (Uri.IsWellFormedUriString(alias, UriKind.Absolute))
+            {
+                throw new CommandExitedException(
+                    "The alias look like a URI to prevent it being ambiguous." +
```

Comment:
```suggestion
                    "The alias should not look like a URI to prevent it being ambiguous." +
```

Reply:
I amended it in 8bc814856ed852ae615571b3db7a70ba554bfa29